<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server Orders</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtSearch">
       <property name="placeholderText">
        <string>Asset Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkCombineOrders">
       <property name="text">
        <string>Combine Orders</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show Trades Grouped By Asset Name</string>
       </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbOrderType">
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lstServerOrders">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnPrev">
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnBulkOrder">
      <property name="text">
        <string>Bulk Execute</string>
      </property>
       <property name="visible">
        <bool>false</bool>
       </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
  
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Prev</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>

  <connection>
   <sender>btnRefresh</sender>
   <signal>clicked()</signal>
   <receiver>actionRefresh</receiver>
   <slot>trigger()</slot>
  </connection>
  <connection>
   <sender>btnPrev</sender>
   <signal>clicked()</signal>
   <receiver>actionPrev</receiver>
   <slot>trigger()</slot>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>actionNext</receiver>
   <slot>trigger()</slot>
  </connection>



  <connection>
   <sender>cmbOrderType</sender>
   <signal>currentTextChanged()</signal>
   <receiver>Dialog</receiver>
   <slot>full_reset()</slot>
  </connection>
  <connection>
   <sender>chkCombineOrders</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>full_reset()</slot>
  </connection>


  <connection>
   <sender>btnBulkOrder</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>bulk_execute()</slot>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>Dialog</receiver>
   <slot>refresh_listings()</slot>
  </connection>
  <connection>
   <sender>actionPrev</sender>
   <signal>triggered()</signal>
   <receiver>Dialog</receiver>
   <slot>prev_page()</slot>
  </connection>
  <connection>
   <sender>actionNext</sender>
   <signal>triggered()</signal>
   <receiver>Dialog</receiver>
   <slot>next_page()</slot>
  </connection>

  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
